src/runner/index.js

```diff
@@ -51,6 +60,28 @@ export default class Runner extends EventEmitter {
         return reporter.testCount - reporter.passed;
     }
 
+    _runTask (reporterPlugin, browserSet, tests) {
+        var task     = new Task(tests, browserSet.connections, this.proxy, this.opts);
+        var reporter = new Reporter(reporterPlugin, task, this.opts.reportOutStream);
+
+        var taskCompleted = false;
+
+        var completeTask = () => taskCompleted = true;
+
+        var completionPromise  = this._getTaskResult(task, browserSet, reporter);
+
```

Comment:
We can remove new line beetwen these single-line declarations and join them with declarations above.